libmachine/log/bugsnag.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/bugsnag/bugsnag-go"
+	"github.com/docker/machine/version"
+)
+
+var metaData bugsnag.MetaData
+
+func Setup(apiKey string) error {
+	if apiKey == "" {
+		apiKey = "e98db82ef639114001cd3397b2c9f885"
```

Comment:
Think we should just make this a package-level `const`? I don't see why any invoker would ever call it with a different value.